Graphs/test/FordFulkerson.test.js

```diff
@@ -0,0 +1,46 @@
+import { fordFulkerson } from '../FordFulkerson.js'
+
+test('Test Case 1', () => {
```

Comment:
1. These should all be in one big `describe("Edmonds-Karp", ...)`.
2. These test case names should be descriptive and useful, or be omitted altogether.
3. How would I easily verify these tests? I think some visualizations of the graphs might be helpful, perhaps as ASCII art, or if these are from an external source, a link to where you got them from.